packages/hardhat/test/Quest.test.js

```diff
@@ -0,0 +1,152 @@
+const { ethers } = require("hardhat");
+const { fail } = require("assert");
+const { use, expect } = require("chai");
+const { solidity } = require("ethereum-waffle");
```

Comment:
I don't understand how and why I should use chain.use :S

Reply:
I don't understand how and why I should use chain.use :S

Reply:
And how I should import expect from chai ?

Reply:
Sorry, I was wrong. We need `ethereum-waffle` package to be installed. But no need to use anything other than the `waffle` object that hardhat injects in the HRE (Hardhat Runtime Environment).

The final code will look something like:
```ts
const { ethers, waffle } = require("hardhat");
const { expect } = waffle // otherwise we can write 'waffle.excpect' directly.
```

What I mean by `chain.use` was that the statement `use(solidity);` is no longer needed. So we can stop importing the `use` method from chai.

Reply:
What I mean by `chain.use` was that the statement `use(solidity);` is no longer needed. So we can stop importing the `use` method from chai.

Reply:
Do you mean we should use assert from mocha instead of expect from chai when we use hardhat-waffle ?

Reply:
![image](https://user-images.githubusercontent.com/13890618/126103262-3f544d4a-67a5-42ff-9ddb-569ebf7de963.png)

Reply:
cause expect doesn't seam to be a function of waffle when I run my tests

Reply:
I also tried it in code but that doesn't work :S

Reply:
I got confused. I thought waffle also had the `expect` logic from chai. Sorry to confuse you as well 😅

Reply:
No worries :p